backend/src/main/java/gov/cdc/usds/simplereport/db/model/TimeOfConsent.java

```diff
@@ -0,0 +1,24 @@
+package gov.cdc.usds.simplereport.db.model;
+
+import javax.persistence.Entity;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+
+@Entity
+public class TimeOfConsent extends EternalAuditedEntity {
```

Comment:
This doesn't seem like the right base class. The EternalAuditedEntity declares that it is
* soft-deletable
* mutable
* created and edited by ApiUsers

Unless consent is revocable, none of these things seems true for this entity (specifically, it appears to be an immutable entity that can only be created by one person, and that person does not authenticate as an ApiUser). 

Now that I think about it, this gives me an enormous misgiving about the mutation operations we currently allow the patient to do on their own test orders—how is that not breaking our audit machinery?

Reply:
You're right - this is not the right base class. In fact, all we really need for this class is a `created_at` column - none of the others, since it's not created by an `ApiUser`. I'll change this.
Regarding the other issue about test order mutation - Adam opened up a ticket on that [here])https://github.com/CDCgov/prime-simplereport/issues/968)